src/components/cards/ProductCard.jsx

```diff
@@ -14,7 +15,9 @@ export function ProductCard({ value }) {
       <div className="card card-vertical">
         {outOfStock && <span className="card-overlay">OUT OF STOCK</span>}
         {badge && <span className="card-badge">{badge}</span>}
-        <i className="far fa-heart"></i>
+        {state.wishlist.filter(item=>item._id===_id).length===1
```

Comment:
instead of filtering and then checking if length = 1, you can do it with find method of array

Reply:
Thanks for the suggestion, I have updated  it.